app/src/main/java/org/listenbrainz/android/data/repository/YimRepository.kt

```diff
@@ -5,4 +5,8 @@ import org.listenbrainz.android.util.Resource
 
 interface YimRepository {
     suspend fun getYimData(username: String): Resource<YimData>
+    
+    fun getUsername(): String?
+    
+    fun getLoginStatus(): Int
```

Comment:
Same with this. Please remove these from the repo

Reply:
Done